rpm/firejail.spec

```diff
@@ -54,6 +45,7 @@ Requires: %{name} = %{version}-%{release}
 %install
 %make_install
 rm -rf %{buildroot}%{_datadir}/bash-completion
+rm -rf %{buildroot}%{_datadir}/zsh/site-functions/_firejail
```

Comment:
Yes, that's why I'm removing that file. I didn't see a way to prevent it from installing and we already remove those bash completion files so it seemed appropriate to remove zsh support files (it's a shell we don't even package in Sailfish OS).

Reply:
Ok, was just looking odd. Thanks for explanation.